probes/src/main/java/cloud/fogbow/probes/core/controllers/DefaultProbesController.java

```diff
@@ -0,0 +1,28 @@
+package cloud.fogbow.probes.core.controllers;
+
+import cloud.fogbow.probes.core.controllers.scheduler.DefaultProbeScheduler;
+import cloud.fogbow.probes.core.controllers.scheduler.ProbeScheduler;
+import cloud.fogbow.probes.core.probes.Probe;
+import cloud.fogbow.probes.core.probes.ProbeCreator;
+import cloud.fogbow.probes.core.probes.docker.creators.DockerProbeCreator;
+import java.util.List;
+
+public class DefaultProbesController {
+
+    private ProbeScheduler probeScheduler;
+    private ProbeCreator probeCreator;
+
+    public DefaultProbesController() {
+        this.probeScheduler = new DefaultProbeScheduler();
+        this.probeCreator = new DockerProbeCreator();
+    }
+
+    public void submitProbes() {
```

Comment:
Where are you submitting the probes?

Reply:
I am sending to the Probe Scheduler that the controller has.